src/HotChocolate/Core/src/Types/Types/Descriptors/ObjectFieldDescriptor.cs

```diff
@@ -60,7 +61,8 @@ public class ObjectFieldDescriptor
 
         if (member is MethodInfo m)
         {
-            Parameters = m.GetParameters().ToDictionary(t => new NameString(t.Name!));
+            _parameterInfos = m.GetParameters();
+            Parameters = _parameterInfos.ToDictionary(t => new NameString(t.Name!));
```

Comment:
I was just wondering, are all valid c# parameter names also valid NameStrings?

Reply:
This might cause issues ... I will open an issue for this and we are gonna fix it in a separate PR